ElectronicObserver/Utility/EOMediaPlayer.cs

```diff
@@ -187,9 +144,9 @@ public List<string> GetPlaylist()
 	public void SetPlaylist(IEnumerable<string> list)
 	{
 		if (list == null)
```

Comment:
Can list actually be null? If yes, then change the parameter type to `IEnumerable<string>?`.